topics/introduction/tutorials/galaxy-intro-short/tutorial.md

```diff
@@ -297,7 +297,7 @@ Where is your first history, called "my-analysis"?
 >
 >    This makes a copy of the dataset in the new history (without actually using additional disk space).
 >
-> 3. Click on **Analyze Data** in the top panel to go back to your analysis window
+> 3. Click on the Home icon {% icon galaxy-home %} (or **Analyze Data** on older versions of Galaxy) in the top panel to go back to your analysis window
```

Comment:
@nomadscientist and the @galaxyproject/training-spanish team, could you make the corresponding change in the Spanish version?

Reply:
Done! https://github.com/galaxyproject/training-material/pull/2698